docs/content/bundle/create.md

```diff
@@ -0,0 +1,200 @@
+You can install tools, certificates, define environment variables, mount secrets, define build arguments, clone repositories, copy files from your local filesystem, and more.
+
+## Customize the Install Action
+
+The default install action uses the exec mixin to call the helpers.sh script in the bundle directory.
+The script prints out "Hello world".
+
+Add an array entry, called a **step**, to the **install** section in porter.yaml and use a mixin to run a command when the bundle is installed.
+The syntax for each mixin is different so reference the documentation for your particular mixin to know what to specify.
+The general form for any mixin is as follows:
+```yaml
+install:
+- exec:
+    description: Optional description of the step
+    # ... mixin specific values
+- terraform:
+    description: Optional description of the step
+    # ... mixin specific values
+```
+
+If you are not using custom mixins, put your bundle's logic in an executable, such as bash script or compiled binary, and place it in the bundle directory (next to your porter.yaml file).
+During porter build, these files are copied into the bundle's image and can be used when the bundle is run.
+You can always use the exec mixin when there isn't an existing mixin, or the mixin doesn't support a particular command that you require.
+
+🙏🏼 Please, [do not embed bash commands] directly in the porter.yaml file because it is much more difficult to get the escaping and quotes correct than putting the bash in a separate file.
+
+Bundles actions should be [idempotent], meaning an action can be repeated without causing errors, and it should have the same effect each time.
+A user of your bundle should be able to re-run install/upgrade/uninstall multiple times in a row, perhaps because the bundle failed half-way through when run the first time.
+Make sure that your commands gracefully handle resources already existing in the install and upgrade actions, and handles already deleted resources in the uninstall action.
+Some mixins, like the exec mixin, have [built-in support for error handling][ignore-errors] and ignoring errors in certain circumstances.
+
+## Test Your Bundle
+
+After you have finished editing the porter.yaml, repeat the `porter build` command to re-build the bundle with your latest changes.
+Then run `porter install mybundle --force` to install the bundle.
+The \--force flag is only safe to use in development, and it allows you to incrementally develop a bundle and re-install it without having to first uninstall it and start over after every change.
+
+## Publish Your Bundle
+
+When you are ready to share your bundle with others, the next step is to publish it to a registry.
+Most registries work with Porter, if you run into trouble check our list of [compatible registries]. 
+
+The [porter publish] command by default pushes the bundle to the registry defined in the porter.yaml file.
```

Comment:
Another unlinked one, I supposed it's should be reference this one (?) https://deploy-preview-2007--porter.netlify.app/cli/porter_publish/